src/sentry/static/sentry/app/components/events/interfaces/threads/threadsSelector.tsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from '@emotion/styled';
+import {ClassNames} from '@emotion/core';
+
+import {Thread} from 'app/types/events';
+import {Event, EntryTypeData} from 'app/types';
+import DropdownAutoComplete from 'app/components/dropdownAutoComplete';
+import DropdownButton from 'app/components/dropdownButton';
+import space from 'app/styles/space';
+import theme from 'app/utils/theme';
+
+import filterThreadInfo from './filter-thread-info';
+import getThreadException from './get-thread-exception';
+import ThreadsSelectorOption from './threadsSelectorOption';
+import ThreadsSelectorSelectedOption from './threadsSelectorSelectedOption';
+
+interface Props {
+  threads: Array<Thread>;
+  activeThread: Thread;
+  event: Event;
+  onChange?: (Thread: Thread) => void;
+}
+
+const DROPDOWN_MAX_HEIGHT = 400;
+
+const ThreadsSelector: React.FC<Props> = ({threads, event, activeThread, onChange}) => {
+  const getDropDownItem = (thread: Thread) => {
+    const threadInfo = filterThreadInfo(thread, event, false);
+
+    const dropDownValue = `#${thread.id}: ${thread.name} ${threadInfo.label} ${threadInfo.filename}`;
+    let crashedInfo: undefined | EntryTypeData = undefined;
```

Comment:
Assigning this `undefined` is a bit redundant

Reply:
yes, I agree with you...but the compile was not satisfied with it yesterday... I also would prefer to only do like this: `return;` let me double check it :)